src/config.rs

```diff
@@ -880,6 +880,8 @@ pub struct DbConfig {
     pub max_total_wal_size: ReadableSize,
     pub max_background_jobs: i32,
     #[config(skip)]
+    pub max_background_flushes: i32,
```

Comment:
no need to have `#[config(skip)]`, since this one is dynamic changeable?

Reply:
@tabokie seems this is not resolved.